.github/workflows/needs-port-upstream.yml

```diff
@@ -0,0 +1,38 @@
+name: needs port upstream
+on:
+  repository_dispatch:
+    types: [port-upstream]
+permissions:
+  issues: write
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: get upstream pull request
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        id: set-result
+        with:
+          script: |
+            const { data } = await github.request('GET /repos/{owner}/{repo}/commits/{commit_sha}/pulls', {
```

Comment:
do you think we could instead pass the pull sha via repository_dispatch action rather than performing another request? (I wonder if two PRs merged quickly might end up creating the same issue.)

Reply:
I think it's possible by changing the trigger from `push` to `pull_request`. 
I'll give it a try.

Reply:
By using `pull_request` as the trigger, I was able to obtain the `html_url`, so I tried passing it directly. What do you think?
https://github.com/wattanx/repository-dispatch-owner/blob/a50893c10af3af35b4f8a73ff0a982b608d99d58/.github/workflows/dispatch-update-file.yml#L22

Reply:
Looks perfect!